### PR TITLE
woeusb: add page

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/woeusb.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# WoeUSB
```
Comment (CleanMachine1) [inline pages/linux/woeusb.md:1]:
```suggestion
# woeusb
```

File: pages/linux/woeusb.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+> Microsoft Windows® USB installation media creator for GNU+Linux.
```
Comment (CleanMachine1) [inline pages/linux/woeusb.md:3]:
```suggestion
> Windows media creation tool.
```

File: pages/linux/woeusb.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://github.com/WoeUSB/WoeUSB>.
+
+- Completely WIPE the entire USB storage device, then build a bootable Windows USB device from scratch:
```
Comment (CleanMachine1) [inline pages/linux/woeusb.md:6]:
```suggestion
- Format a USB then create a bootable Windows installation drive:
```

File: pages/linux/woeusb.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+`woeusb --device {{source media path}} {{device}}`
```
Comment (CleanMachine1) [inline pages/linux/woeusb.md:8]:
```suggestion
`woeusb --device {{path/to/windows.iso}} {{/dev/sdX}}`
```

File: pages/linux/woeusb.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+- Copy Windows files to an existing partition of a USB storage device and make it bootable. This allows files to coexist as long as no filename conflict exists:
```
Comment (CleanMachine1) [inline pages/linux/woeusb.md:10]:
```suggestion
- Copy Windows files to an existing partition of a USB storage device and make it bootable, without erasing the current data:
```

File: pages/linux/woeusb.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+`woeusb --partition {{source media path}} {{partition}}`
```
Comment (CleanMachine1) [inline pages/linux/woeusb.md:12]:
```suggestion
`woeusb --partition {{path/to/windows.iso}} {{/dev/sdXN}}`
```

**Comment by suyashbhawsar** [comment]
Hello, is there something else I have to do? I'm contributing to the tldr repo for the first time.

**Comment by CleanMachine1** [comment]
Hi, yes. 

As a contributor, you commit the suggestions which you agree with. Then any other ones which you think could be improved, we can discuss.

**Comment by suyashbhawsar** [comment]
> As a contributor, you commit the suggestions which you agree with.

All the suggestions you provided looks good, as it makes the the documentation simpler to read. So, how do I commit the changes?

**Comment by CleanMachine1** [comment]
@suyashbhawsar You seem to have added another page to this PR, was this intentional?

**Comment by suyashbhawsar** [comment]
> @suyashbhawsar You seem to have added another page to this PR, was this intentional?

@CleanMachine1 Actually, I wanted to create another pull request for that but, it automatically got added in this one. Is there a way to create a separate pull request while having one open?

**Comment by CleanMachine1** [comment]
Remove the page from this branch.
Make a new branch based from main.
Now add that page to the new branch, push to remote and make the PR.
Hope that helps

**Comment by suyashbhawsar** [comment]
@kbdharun Can you check this pull request too?

**Review by kbdharun — APPROVED** [review]
LGTM

**Review by CleanMachine1 — APPROVED** [review]

**Comment by suyashbhawsar** [comment]
@CleanMachine1 Thanks! I created a separate Pull request for `broot`. I'd appreciate if you give it a check:
https://github.com/tldr-pages/tldr/pull/8487

**Pull request merged**